JsBasics/Objects/task2.js

```diff
@@ -0,0 +1,13 @@
+﻿
+
+const book = {
+    title: "Wonderful moment",
+    author: "Maria Ride",
+    year: 2004
+
+}
+const { title, author, year } = book;
```

Comment:
Знову таки, читаємо завдання уважніше
```suggestion
const { title, author} = book;
```